cluster/src/dunit/scala/io/snappydata/cluster/QueryRoutingDUnitTest.scala

```diff
@@ -337,12 +354,12 @@ class QueryRoutingDUnitTest(val s: String) extends ClusterManagerTestBase(s) {
     }
     assert(foundTable)
 
-    val rSet2 = dbmd.getTables(null, JDBCAppendableRelation.INTERNAL_SCHEMA_NAME, null,
+    val rSet2 = dbmd.getTables(null, INTERNAL_SCHEMA_NAME + "APP", null,
```

Comment:
If user  provides fully qualified tableName then there was a inconsistency in the product while creating the internal column table name and retrieving the row buffer name out of this.

This pull request also fixes that bug by using the naming  new convention  INTERNAL_SCHEMA_NAME + "fully qualifuied tableName" +  SHADOW_TABLE_SUFFIX.
This change done to support the default schema as well.